Python/py_solutions_61-70/Euler_61.py

```diff
@@ -62,13 +67,23 @@ def euler_61():
     heptagon = lambda n: int(n * (5 * n - 3) / 2)
     octagon  = lambda n: n * (3 * n - 2)
     polygonal_four_digit = partial(create_four_digit_polygonal_set, 999, 10000)
-    cyclic = list(map(polygonal_four_digit, [triangle, square, pentagon,
-                                             hexagon, heptagon, octagon]))
-    return sum(find_cyclic_set(cyclic[0][0], [], cyclic, 0))
+    cyclical_sets = list(map(polygonal_four_digit, [triangle, square, pentagon,
```

Comment:
This is where the updated algorithm picks-up.  Having a hash for each polygonal number created to map which number is linked to others and what their base-sequence function is.